distribution/docs/src/main/resources/_contents/_plugins/metacard-backup-plugin-contents.adoc

```diff
@@ -0,0 +1,24 @@
+==== Metacard Backup Plugin
+
+The Metacard Backup Plugin is used to enable data backup of metacards using a configurable transformer.
+The plugin will store backups using a configured Metacard Backup Storage provider.
+
+===== Installing the Metacard Backup Plugin
+
+To install the Metacard Backup Plugin
+
+. Navigate to the *${admin-console}*.
+. Select ${ddf-catalog} application.
+. Select *Features* tab.
+. Install the `catalog-metacard-backup` feature.
+
+===== Configuring the Metacard Backup Plugin
+
+To configure the Metacard Backup Plugin:
```

Comment:
This documentation is to configure the Metacard Backup Plugin itself. There are 2 docs. One for the plugin and one for the storage provider. No documentation existed from before, so I added it.